netjsonconfig/backends/raspbian/templates/interfaces.jinja2

```diff
@@ -0,0 +1,83 @@
+{% for i, j in data.items() %}
+{% if i|string() == 'interfaces' %}
+{% for interface in j %}
+{% if interface.get('iftype') == 'wireless' %}
+{% if interface.get('mode') == 'adhoc' %}
+config: /etc/network/interfaces
```

Comment:
are you sure this line doesn't make the configuration invalid when deployed on an actual device?

Shouldn't it be `# config: /etc/network/interfaces`?